api/typeDefs/notifications.js

```diff
@@ -95,6 +95,7 @@ export default gql`
     id: ID!
     earnedSats: Int!
     sortTime: Date!
+    withdrawl: Withdrawl!
```

Comment:
This is backwards compatible because it's a new field that is mandatory. Old code is not querying it.

If I would change an existing type from optional to mandatory, it would not be backwards compatible.